sklearn/metrics/tests/test_pairwise.py

```diff
@@ -415,6 +416,22 @@ def test_rbf_kernel():
     assert_array_almost_equal(K.flat[::6], np.ones(5))
 
 
+def test_matern_kernel():
```

Comment:
All if branches are now tested (line 437ff). The absolute exponential test (line 430ff) is a bit self-referential; an alternative would be to test against the absolute_exponential in gaussian_process.correlation_models.